### PR TITLE
US-389592: Provide rate limit configuration to search service - SRS mock

### PR DESCRIPTION
**Pull request by andjelx**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
 PATH_REQUEST = '/request'
 PATH_TCP = '/tcp'
 PATH_POSTGRES = '/postgres'
+PATH_SEARCH = '/search/'
```
Comment (malmanzor) [inline server.py]: Does it make sense to use a more generic name?

File: server.py [hunk 2]
```diff
@@ -171,6 +172,21 @@ def testpostgres():
...
 
+# Search service
+@app.route(PATH_SEARCH)
+@app.route(PATH_SEARCH + '<string:tenant_id>/')
+def search_url_get(tenant_id=None):
+    response_code = request.args.get(RESPONSECODE, default=200, type=int)
+    return json.dumps({RESPONSECODE: response_code, "tenantID": tenant_id}), response_code
+
+
+@app.route(PATH_SEARCH + '<string:tenant_id>/model', methods=['POST'])
```
Comment (malmanzor) [inline server.py]: Does it make sense to make the entire suffix flexible?

File: server.py [hunk 3]
```diff
@@ -171,6 +172,21 @@ def testpostgres():
...
+@app.route(PATH_SEARCH + '<string:tenant_id>/index', methods=['POST'])
+@app.route(PATH_SEARCH + '<string:tenant_id>/query', methods=['POST'])
+def search_url_post(tenant_id):
+    return json.dumps({"Status": "Ok", "tenantID": tenant_id}), 200
```
Comment (malmanzor) [inline server.py]: For testing purposes, it would be good to include the action (index/query/model) as part of the response

**Comment by andjelx** [comment]
@malmanzor Updated the code - took into account all your remarks.

**Review by bachirgit — APPROVED** [review]

**Review by malmanzor — APPROVED** [review]

**Pull request merged**